controllers/vcdmachine_controller.go

```diff
@@ -592,6 +588,63 @@ func (r *VCDMachineReconciler) getOVDCDetailsForMachine(ctx context.Context, mac
 	}
 }
 
+func GetMachineDeploymentName(ctx context.Context, cli client.Client, vcdCluster *infrav1beta3.VCDCluster,
```

Comment:
Can we call this function `GetNodePoolName()` ? because it either returns the kcp name or machine deployment name.

Reply:
yes, we can